### ISSUE TITLE
integrationTest mistakenly run during gradle check

### ISSUE THREAD

**Issue by mcandre** [unbroken-dome/gradle-testsets-plugin#16]
I've added an integrationTest set, which works as expected with `gradle test` (just the unit tests) and `gradle integrationTests` (unit + integration tests).

However, when I run `gradle check`, Gradle executes integration tests, which I don't want to happen.

build.gradle:

```
  testSets {
    integrationTest { dirName = 'integration-test' }
  }
  integrationTest.dependsOn test
```

What is some configuration I can use to work around this? Is there a way to disconnect the integrationTest task from the check task?

**Comment by JasonKDarby** [comment]
It's because [check runs test](https://docs.gradle.org/current/userguide/java_plugin.html).  You can disconnect it from the test task, but otherwise I'd just suggest making a new task that runs test and integrationTest.

**Comment by mcandre** [comment]
Thanks! How can I disconnect the integrationTest task from the check stask?

**Comment by JasonKDarby** [comment]
I meant that if you remove `integrationTest.dependsOn test` then `check` won't call integrationTest anymore because `test` won't call `integrationTest` anymore.

**Comment by mcandre** [comment]
Is my dependency syntax wrong? In plain English terms, I want integration tests to depend on unit tests, not unit tests depending on integration tests.

**Comment by JasonKDarby** [comment]
Whoops, I had it backwards.  I'm not sure why check calls it.  I'll see if I can reproduce it later.

**Comment by tkrullmann** [comment]
I know this has been open a while... I cannot reproduce it with any version of Gradle or the plugin.
Is this still an issue?
